internal/controller/persistentvolume/persistentvolume.go

```diff
@@ -122,6 +122,40 @@ func checkStaticVolume(pv *corev1.PersistentVolume) (bool, error) {
 	return static, nil
 }
 
+// storeVolumeIDInPV stores the new volumeID in PV object.
+func (r ReconcilePersistentVolume) storeVolumeIDInPV(pv *corev1.PersistentVolume, newVolumeID string) error {
```

Comment:
There probably should be a verification that the PV is a Ceph-CSI one?

Reply:
The check already exists. the controller will reconcile the only PV created for this Driver.